ci-operator/step-registry/hypershift/mce/heterogeneous/OWNERS

```diff
@@ -0,0 +1,15 @@
+approvers:
```

Comment:
I think this folder might cause some confusion because these are worker nodes created through the agent for various architectures. I believe `mce/ibm` or `mce/ibm/heterogeneous` might be better choices.

Reply:
Thanks for the review!
The reason why we kept it outside the power folder was, so that it could be reused by other combinations of heterogeneous workflows of other architectures which are not power specific and could be used by other teams.
But, if you feel this approach might not be as effective, we can change it to  `hypershift/mce/power/heterogeneous`

Reply:
I can confirm that no other directories will use the power or ibmz step, so I believe we can move to `/hypershift/mce/power`.